src/thunderbird_accounts/urls.py

```diff
@@ -24,16 +24,14 @@
     path('auth/<str:login_code>/?redirect_to=<str:redirect_to>', auth_views.fxa_start, name='fxa_login'),
     # This will be auth/callback in the future.
     re_path(r'^favicon\.ico$', favicon_view),
-
     # Test
     path('', mail_views.home),
     path('self-serve', mail_views.self_serve),
-
+    path('waitlist', mail_views.waitlist),
```

Comment:
```suggestion
    path('wait-list', mail_views.wait_list),
```

makes it a little easier to read